.github/workflows/_sys_test.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_call:
+      inputs:
+        python-version:
+          type: string
+          description: The version of python to install
+          required: true
+        runs-on:
+          type: string
+          description: The runner to run this job on
+          required: true
+
+
+jobs:
+  run:
+    runs-on: ${{ inputs.runs-on }}
+    env:
+      EC_INTERACTIVE_TESTING: "true"
+      EC_K8S_NAMESPACE: "default"
+      EC_SERVICES_REPO: "https://github.com/epics-containers/bl47p.git"
+    steps:
+      - name: start minikube
+        id: minikube
+        uses: medyagh/setup-minikube@master
```

Comment:
please pin the unofficial github action to avoid supply chain attacks.

e.g. https://github.com/epics-containers/edge-containers-cli/blob/7054914012871926c3c740b66ee241510e2eb38f/.github/workflows/_release.yml#L26